arlo-client/src/components/Header.tsx

```diff
@@ -36,6 +50,27 @@ const Header: React.FC<{}> = () => {
         </NavbarHeading>
       </NavbarGroup>
       <NavbarGroup align={Alignment.RIGHT}>
+        {isAuthenticated && electionId && meta!.type === 'audit_admin' && (
```

Comment:
enum / constant here too for `audit_admin`.

Reply:
@benadida It's already constrained by the type structure of the values returned from `useAuthDataContext` to the two allowed values, and that covers this here too. Is that adequate for what you're thinking?

Reply:
Yeah, I can live with that.